### ISSUE TITLE
JS help needed

### ISSUE THREAD

**Issue by liach** [FabricMC/MappingPoet#8]
I want to make the mapping table items copied to clipboard on click. However, I currently have some trouble doing that. Seeking help!

**Comment by altrisi** [comment]
Is this what you are looking for? (or something similar) (based on the JS file in the repo)

This will copy any `span` element when clicked by selecting it, copying it, and then removing that selection.

```js
const items = document.querySelectorAll("span");

items.forEach(item => {
  item.onclick = function() {
  	var range = document.createRange();
  	range.selectNode(item);
  	window.getSelection().addRange(range);

    var result = document.execCommand('copy');
    console.log(result ? 'Copied to clipboard' : 'Couldn\'t copy'); //Completely optional

    window.getSelection().removeRange(range);
  };
});
```

Sorry if it's not, I've not been able to run this repo to see how is this integrated (tried building alone and creating javadocs with yarn PR, all errored. Not sure if that's the way). It does however do what (I think) that snippet tries to do (first part).

This is not perfect: It will clear selections (aka copy but not let the user select) if the user was selecting the element being copied when clicking. Also, if the user had something selected and clicks the item while adding to the selection (Ctrl+Drag), then both will be copied. But it works in general. To solve that, the current selection could be exported to a variable, then cleared, then this code, then readded.

**Comment by altrisi** [comment]
Wait, I think I've got it.

**Comment by liach** [comment]
Should keep this open until I verify now copy on click works well

**Comment by liach** [comment]
Should be addressed in 0.2.2.
